docs/tutorial/debugging-main-process-vscode.md

```diff
@@ -35,3 +39,49 @@ $ code electron-quick-start
 Set some breakpoints in `main.js`, and start debugging in the [Debug View](https://code.visualstudio.com/docs/editor/debugging). You should be able to hit the breakpoints.
 
 Here is a pre-configured project that you can download and directly debug in VSCode: https://github.com/octref/vscode-electron-debug/tree/master/electron-quick-start
+
+## Native C++ (Windows)
+
+### 1. Open an Electron project in VSCode.
+
+```sh
+$ git clone git@github.com:electron/electron-quick-start.git
+$ code electron-quick-start
+```
+
+### 2. Add a file `.vscode/launch.json` with the following configuration:
+
+```json
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "(Windows) Launch",
+      "type": "cppvsdbg",
+      "request": "launch",
+      "program": "${workspaceFolder}\\out\\Testing\\electron.exe",
+      "args": [],
+      "stopAtEntry": false,
+      "cwd": "${workspaceFolder}",
+      "environment": [
+          {"name": "ELECTRON_ENABLE_LOGGING", "value": "true"},
+          {"name": "ELECTRON_ENABLE_STACK_DUMPING", "value": "true"},
+          {"name": "ELECTRON_RUN_AS_NODE", "value": ""},
+      ],
+      "externalConsole": false,
+      "sourceFileMap": {
+          "o:\\": "${workspaceFolder}",
+      },
+    },
+  ]
+}
+```
+**Note:** `args` should also have one parameter, either the path to the folder or `main.js` file of the Electron project you are using for testing. In this example, it should be your path to `electron-quick-start`.
```

Comment:
These details definitely clear things up. For the part where you switched "using for testing" to "debugging", I initially used this wording since in this section you are debugging the Electron codebase rather than your Electron project. Does that make sense in the context of the file, or does "using for testing" still seem unclear?

Reply:
Ah, `using for testing` makes sense in that context! I've edited my original suggestion in accordance.